server/prisma-rs/prisma-models/src/relation.rs

```diff
@@ -104,7 +104,7 @@ impl RelationTemplate {
         };
 
         let model_a = {
-            let model = relation.with_schema(|schema| schema.find_model(&model_a_name).unwrap());
+            let model = relation.schema().find_model(&model_a_name).unwrap();
```

Comment:
I think this is easier to read than the version where you have to provide a function.

Reply:
It is, was planning to remove these tomorrow...